Resources/Prototypes/Entities/Structures/Doors/Windoors/assembly.yml

```diff
@@ -0,0 +1,80 @@
+﻿- type: entity
+  id: WindoorAssembly
+  name: windoor assembly
+  description: It opens, it closes, and you can see through it!
+  parent: BaseStructure
+  components:
+  - type: InteractionOutline
+  - type: Sprite
+    netsync: false
+    drawdepth: FloorObjects
+    sprite: Structures/Doors/Windoors/windoor.rsi
+    layers:
+      - state: assembly
+      - state: panel_open
+        visible: false
+  - type: Physics
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+        bounds: "-0.2,-0.49,-0.49,0.49"
+      mass: 30
+      mask:
+      - Impassable
+      - VaultImpassable
+  - type: Anchorable
+  - type: Pullable
+  - type: Rotatable
+  - type: Damageable
+    resistances: metallicResistances
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 300
+      behaviors:
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MetalRod:
```

Comment:
Since the door is made with steel sheets and secure with plasteel, it should probably drop those instead of metal rods.